pkgs/development/python-modules/pysam/default.nix

```diff
@@ -27,17 +28,49 @@ buildPythonPackage rec {
   };
 
   nativeBuildInputs = [ samtools ];
-  buildInputs = [ bzip2 curl cython lzma zlib ];
+  buildInputs = [
+    bzip2
+    curl
+    cython
+    libdeflate
+    lzma
+    zlib
+  ];
 
   checkInputs = [ pytest bcftools htslib ];
-  checkPhase = "py.test";
 
-  # tests require samtools<=1.9
-  doCheck = false;
-  preCheck = ''
+  # # tests require samtools<=1.9
+  # doCheck = false;
```

Comment:
Can this be fully removed with the 3 merge requests?

Reply:
Sorry, leftovers from testing. Yes, it will.

Reply:
To be precise, `samtools <= 1.9` still applies, but we are skipping the failing tests which are known issues upstream, due to be fixed in a future release.